src/net.h

```diff
@@ -413,6 +426,12 @@ class CConnman
     std::thread threadOpenAddedConnections;
     std::thread threadOpenConnections;
     std::thread threadMessageHandler;
+
+    /** flag for deciding to connect to an extra outbound peer,
+     *  in excess of nMaxOutbound
+     *  This takes the place of a feeler connection */
+    CCriticalSection m_cs_outbound_peer;
```

Comment:
If this CS doesn't protect more than a single boolean, it's better to instead just use an `std::atomic<bool>`.

Reply:
Done